### PR TITLE
adicionado comando uepa e /comandos

### PR DESCRIPTION
**Pull request by gcoutini**
adicionado comando uepa e /comandos

### CODE REVIEW

File: handler/message/index.ts [hunk 1]
```diff
@@ -20,7 +21,7 @@ export class ZapError extends Error {
 
 export default async (client: Client, message) => {
     try {
-        let { type, id, from, fromMe, to, t, sender, isGroupMsg, chat, caption, isMedia, isGif, mimetype, quotedMsg, quotedMsgObj, mentionedJidList } = message
+        let { type, id, from, fromMe, to, t, sender, isGroupMsg, chat, caption, isMedia, isGif, mimetype, quotedMsg, quotedMsgObj, mentionedJidList, chatId } = message
```
Comment (rafaelgalani) [inline handler/message/index.ts:24]: Why is that? Can't `from` or `id` be used?

File: handler/message/index.ts [hunk 2]
```diff
@@ -125,174 +126,6 @@ export default async (client: Client, message) => {
...
-        //     for (let i = 0; i < zoneStatus.data.length; i++) {
-        //         const { zone, region } = zoneStatus.data[i]
-        //         const _zone = zone == 'green' ? 'Hijau* (Aman) \n' : zone == 'yellow' ? 'Kuning* (Waspada) \n' : 'Merah* (Bahaya) \n'
-        //         data += `${i + 1}. Kel. *${region}* Berstatus *Zona ${_zone}`
-        //     }
-        //     const text = `*CEK LOKASI PENYEBARAN COVID-19*\nHasil pemeriksaan dari lokasi yang anda kirim adalah *${zoneStatus.status}* ${zoneStatus.optional}\n\nInformasi lokasi terdampak disekitar anda:\n${data}`
-        //     client.sendText(from, text)
-        //     break
-        // Group Commands (group admin only)
-        case 'ban':
```
Comment (rafaelgalani) [inline handler/message/index.ts:295]: Wouldn't be better to leave these in a separate commit? 
Although it's basically just old code commented. Might leave it there, I guess.

File: handler/message/index.ts [hunk 3]
```diff
@@ -388,6 +221,12 @@ export default async (client: Client, message) => {
             await client.sendText(groupId, 'Blz kkkjjjj.');
             return await client.removeParticipant(groupId, actor);
         }
+        case 'uepa':{
+            return await client.sendFile(chatId, path.resolve(__dirname, '../../assets/audios/uepa.mp3'), 'uepa', '', '', false, true, false);
```
Comment (rafaelgalani) [inline handler/message/index.ts:225]: Using `'../../'` and `/` as path separator might be considered code smell: a OS might use a path separator different than the conventional slash; same for the relative paths (`'..'`), which could be causing further problems if we later decide to move this file. For that, Node provides some useful methods in the `path` module, such as `resolve()` and `join()`.

File: index.ts [hunk 4]
```diff
@@ -6,17 +6,14 @@ const start = (client = new Client(void 0, void 0, void 0)) => {
...
-
     // Force it to keep the current session
     client.onStateChanged((state) => {
         console.log('[Client State]', state)
         if (state === 'CONFLICT' || state === 'DISCONNECTED') client.forceRefocus()
     })
 
     // listening on message
-    client.onMessage((message) => {
+    client.onAnyMessage((message) => {
```
Comment (rafaelgalani) [inline index.ts:16]: Yeah, about that: this version is outdated. It would be better to update your fork, but that's fine.

**Review by rafaelgalani — CHANGES_REQUESTED** [review]
Just address the following comments and it will be merged.

File: handler/message/index.ts (hunk 1 above)
Comment (gcoutini) [reply handler/message/index.ts:24]: I've just followed the sendFile doc as in https://docs.openwa.dev/classes/client.html#sendfile

File: handler/message/index.ts (hunk 2 above)
Comment (gcoutini) [reply handler/message/index.ts:295]: Thought it was just polluting and decided to clean it up.
